tests/returntypes/test_parameter_shift_shot_vec_new.py

```diff
@@ -1471,18 +1505,18 @@ def test_involutory_and_noninvolutory_variance_single_param(self, tol):
                 assert isinstance(param_res, np.ndarray)
                 assert param_res.shape == ()
 
-            assert shot_vec_result[0] == pytest.approx(expected[0], abs=herm_shot_vec_tol)
-            assert shot_vec_result[1] == pytest.approx(expected[1], abs=herm_shot_vec_tol)
+            assert shot_vec_result[0] == pytest.approx(expected[0], abs=finite_diff_tol)
+            assert shot_vec_result[1] == pytest.approx(expected[1], abs=0.3)
```

Comment:
I think you defined 0.3 as variable before.

Reply:
Not in this file.